Update howtos for cluster-scaling after UI changes (#57)
crate/cloud-howtos: docs/scale-cluster.rst

   :local:

.. _scale-cluster-preferences:

Find the Cluster Preferences page
=================================

> To scale your clusters in the Console, you must find the Cluster Preferences page.

Reviewer: perhaps it would read better if we wrote

```suggestion
To scale your clusters in the Console, navigate to the Cluster Preferences page.
```

Author: fixed

In the menu on the left hand you should see below the divider line an icon or
icons representing all clusters associated with the currently selected project.
Here, find the correct cluster and click on the icon. It will expand and show
links for four pages: *Overview*, *Backups*, *Metrics* and *Preferences*.

.. image:: _assets/img/cluster-dropdown.png
   :alt: Cloud Console projects cluster selection

> Click on *Preferences*. This will take you to Cluster access page, where you

Author: fixed

> can see and edit your credentials. You can also enable Deletion protection 

Author: fixed

> of your cluster and create Whitelist of IP addresses or CIDR blocks that are

Author: fixed

allowed to connect to the cluster. 

.. image:: _assets/img/cluster-access.png
   :alt: Cloud Console cluster access settings

> To edit Cluster scale, click on *Cluster scale* in the top bar. This will take

Author: fixed

> you to a page where you can see overview of your Cluster plan as well as 

Author: fixed

current cluster configuration.

.. image:: _assets/img/cluster-preferences.png
   :alt: Cloud Console cluster preferences

.. _scale-cluster-instructions:

Scale a cluster
===============

> Once arrived at the Cluster scale page, you can scale the cluster you

Reviewer: ```suggestion
Once you arrive at the Cluster scale page, you can scale the cluster you
selected in three steps. First, click the *Edit cluster configuration* button
in the top right.
```

Author: fixed

selected in three steps. First, click the *Edit cluster configuration* button
in the top right.

Secondly, choose the desired scaling value in either the *Storage* or

> *Choose number of nodes* sections.

Reviewer 2: Open question: would we call custom storage increases scaling, in the same way we call expanding nodes scaling? Do we need to distinguish horizontal and vertical? I'm not really sure but wonder what you think

Author: Good question. It probably could be better called "storage increase" to be simpler. But when someone says "storage scaling" I pretty much imagine the same thing: increasing the storage. To keep it uniform, I'd stick with "scaling". Opinions?

Reviewer: no strong opinions. scaling to me represents adding additional nodes to a cluster and has a different mental model to adding storage. I don't know the precise technicalities of increasing storage but I imagine it as a pool of available disk space as opposed to adding additional hard drives, which 'scaling' might suggest. FWIW to me storage increase is the more natural convention.

In the menu on the left hand you should see below the divider line an icon or
icons representing all clusters associated with the currently selected project.
Here, find the correct cluster and click on the icon. It will expand and show
links for four pages: *Overview*, *Backups*, *Metrics* and *Preferences*.

.. image:: _assets/img/cluster-dropdown.png
   :alt: Cloud Console projects cluster selection

Click on *Preferences*. This will take you to cluster access page, where you
can see and edit your credentials. You can also enable deletion protection 
of your cluster and create a whitelist of IP addresses or CIDR blocks that are
allowed to connect to the cluster. 

.. image:: _assets/img/cluster-access.png
   :alt: Cloud Console cluster access settings

To edit the cluster scale, click on *Cluster scale* in the top bar. This will take

> you to a page where you can see an overview of your Cluster plan as well as 

Reviewer 2: cluster

current cluster configuration.

.. image:: _assets/img/cluster-preferences.png
   :alt: Cloud Console cluster preferences

.. _scale-cluster-instructions:

Scale a cluster
===============

> Once you arrive at the Cluster scale page, you can scale the cluster you

Reviewer 2: I suppose we should be consistent about how to refer to the docs tabs/pages/etc... I think in some of these older files I didn't quite do that (e.g., see the bit on Cluster Preferences above, which is a page rather than a tab - what to do?) I think the Console overview in the Reference has an implicit standardization that we should use, might be worth checking out to see if that makes sense to you

The values corresponding to each scale unit will vary depending on the plan
selected for the cluster. This also affects the cluster configuration and
pricing. A warning will appear to remind the user that changing scale settings
will be reflected in the charges billed. You will be able to see the new

> prices of *Compute* and *Storage* in the bottom right.

Reviewer 2: isn't this more 'right' than 'bottom right' at that point? Terribly pedantic though so feel free to ignore me :)
